src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: New runtime transition guide
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+Synthetic monitoring has a new runtime that affects your public monitors. By converting from the legacy runtime to the new runtime, you improve the backend environment that your public monitors execute from. New synthetic monitors default to using the new runtime. If you've created a synthetic monitor using the legacy runtime, there are extra [configuration steps](#runtime-convert).
+
+Making the switch gives you these features:
+
+* Access to newer versions of Google Chrome, so your scripted browser monitors use browser versions similar to the ones used by your customers. 
+* Access to the updated scripted API runtime that uses `Got` node.js library  instead of the deprecated `Request` library. 
+* Support for async / await syntax. This simplifies script creation and maintenance, diverging from the old runtime’s `.then` promise chaining.
+* Better support for shadow DOM objects.
+* [Custom timing library](/docs/synthetics/synthetic-monitoring/scripting-monitors/custom-timing-details) for scripted API monitors.
+- Access to  our [NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial) to automate the management of your synthetic monitors. 
+
+We recommend updating your synthetic monitors that were created using the legacy runtime to take advantage of these capabilities. We plan to release this new runtime for private monitoring locations later this year.
+
+<Callout variant="important">
+Before switching, make sure that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Convert your monitors to the new runtime [#runtime-convert]
+
+Existing monitors default to the runtime used at their time of creation. To prevent your critical monitors from breaking during future end of life processes, we recommend converting your public monitors sooner rather than later. To convert:
```

Comment:
Maybe we should reference the benefits of converting to the new runtime here in addition to referencing future end of life processes?